### ISSUE TITLE
Hose holder

### ISSUE THREAD

**Issue by Silas770** [stijnwop/hoseSystem#11]
A holder where you can attach the tanker side of the hose to while not pumping, so it don't falls into the pit.

May as a prefab and as a placeable object. A buyable dynamic object would also be interesting, but sure it would fall to the side every time, you attach or detach the hose.

**Comment by stijnwop** [comment]
The holder is placed next to the pit right?

Could you add some reference images so I don't forget the whole idea?

**Comment by Silas770** [comment]
Yes. I just found one video where you can see something like I meant.

https://www.youtube.com/watch?v=pENiTFz0JZY&feature=youtu.be&t=85
